### PR TITLE
Updated the jumbo-jekyll-theme to 5.5.1 - abstracting HTML from content pages

### PR DESCRIPTION
**Pull request by kylekirkby**
Updated the jumbo-jekyll-theme to 5.5.1 - abstracting HTML from content pages
- Content pages contain just markdown and frontmatter

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -25,4 +25,6 @@ exclude:
 
 plugins:
     - jekyll-minimagick
-    - mini_magick
\ No newline at end of file
+    - mini_magick
+  
+include: ['_pages']
```
Comment (morancj) [inline _config.yml]:
```suggestion
include: ['_pages']

```

File: _data/footer.yml [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
+      - name: 96Boards
+        url: https://www.96boards.org
+      - name: Linaro Connect
+        url: https://connect.linaro.org
+      - name: DeviceTree
+        url: https://www.devicetree.org
+      - name: OP-TEE
+        url: https://www.op-tee.org
+      - name: Linaro Developer Cloud
+        url: https://www.linaro.cloud
```
Comment (morancj) [inline _data/footer.yml]:
```suggestion
        url: https://www.linaro.cloud

```

File: _data/sticky-tab-bar.yml [hunk 3]
```diff
@@ -12,13 +12,4 @@ pages:
...
-    list:
-      -
-        title: Documentation
-        url: /documentation/
-      -
-        title: License Agreement
-        url: /agreement/
-    urls: [/documentation/, /agreement/]
\ No newline at end of file
+    urls: [/governance/,/releases/,/committee/]
```
Comment (morancj) [inline _data/sticky-tab-bar.yml]:
```suggestion
    urls: [/governance/,/releases/,/committee/]

```

File: _includes/sticky-tab-bar.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                </div>
+            </div>
+        </nav>
+    </div>
+</div>
+{% elsif custom-include %}
+    {% include {{custom-include}}.html %}
+{% else %}
+No sticky-nav.yml content found - please add to _data/sticky-tab-bar.yml. More information available <a href="/jekyll/includes/sticky-tab-bar/">here</a>.
+{% endif %}
```
Comment (morancj) [inline _includes/sticky-tab-bar.html]:
```suggestion
{% endif %}

```

File: assets/css/app/custom.scss [hunk 5]
```diff
@@ -149,4 +149,10 @@ div#contact-thanks {
 }
 #jumbotron-content img {
     height: 200px;
+}
+div#flow_wrapper #content-container {
+    padding-bottom: 0px;
+}
+#flow_wrapper div#main_content {
+    padding-top: 0px;
 }
```
Comment (morancj) [inline assets/css/app/custom.scss]:
```suggestion
}

```

File: assets/css/app/overrides.scss [hunk 6]
```diff
@@ -54,3 +54,5 @@ $post-tag-hover-bg: #1f1f1f !default;
 $post-tag-hover-color: #fff !default;
 // Triangle Divider
 $triangle-divider-bg-color: #fff !default;  
+// Flow sections
+$flow_row_padding: 40px !default;
```
Comment (morancj) [inline assets/css/app/overrides.scss]:
```suggestion
$flow_row_padding: 40px !default;

```

File: .gitignore [hunk 7]
```diff
@@ -4,3 +4,4 @@
 _site
 production.devicetree.org
 staging.devicetree.org
+Gemfile.lock
```
Comment (morancj) [inline .gitignore]:
```suggestion
Gemfile.lock

```

**Comment by kylekirkby** [comment]
@morancj - thanks for the review. All suggested changes should be now added to the PR.

**Pull request merged**